compiler/rustc_hir_analysis/src/collect/resolve_bound_vars.rs

```diff
@@ -1068,13 +1072,13 @@ impl<'a, 'tcx> Visitor<'tcx> for BoundVarContext<'a, 'tcx> {
             GenericParamKind::Lifetime { .. } => {}
```

Comment:
Is this match any different than just calling walk_generic_param? seems a bit more verbose than other visitors lol

Reply:
I think this is just an inlined `walk_generic_param` yeah...